app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
 
   SSO_ATTRIBUTES = %w[admin avatar_url banned custom_fields email name username].freeze
 
-  has_one :subscription
+  has_one :subscription, -> { where active: true }
```

Comment:
Why is this needed?

Reply:
Because you can have a previous plan that's inactive, and get a new one that's different from that one that will be active.

Reply:
Ok, see my comment below